### PR TITLE
Account 2FA enforcement article

### PR DESCRIPTION
**Pull request by OleMchls**
This article covers all questions around 2FA enforcement for shared accounts.

### CODE REVIEW

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Enforce Two-Factor Authentication for a shared Account
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]: I suggest

> Enforce Two-Factor Authentication for all members of an account

The term _shared_ is ambiguous as we use it for sharing a domain. It's not really a shared account, it's just an account with multiple members.

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+excerpt: How to protect your shared DNSimple account using Two-Factor authentication enforcement.
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]:
```suggestion
excerpt: How to ensure an account is secured by enforcing all the members of an account to use two-factor authentication.
```

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+</info>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+DNSimple supports [Two-Factor authentication](/articles/two-factor-authentication/). This feature enforces that all users in our account must have Two-Factor authentication enabled.
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]:
```suggestion
[Two-Factor authentication](/articles/two-factor-authentication/) provides an extra level of security to protect your user credentials. You can enforce account members to use two-factor authentication by enabling the feature for the account.
```

I replaced the concept of _your_ account as it seems you own an account. Whereas, in fact, the account is not owned by anyone, but members belong to an account.

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+When you have added other users to your account with it is important to ensure the security of each account member. This is why this feature is available to all plans that have access to the teams feature.
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]: This sentence is quite confusing. I can't understand what it means. Also, please let's not refer as the Team feature.

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+## Enabling Two-Factor authentication enforcement {#enable}
+
+You can enable Two-Factor authentication enforcement from your account members page.
+
+You can only enable Two-Factor authentication enforcement for your account, if all users have Two-Factor authentication enabled. You either need to bug them to enable it or remove them from the team to enable this feature.
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]: Removing _your_ account. Users have multiple accounts, and they don't are the unique owners.

```suggestion
You can only enable Two-Factor authentication enforcement an account only if all users have Two-Factor authentication enabled. You either need to bug them to enable it or remove them from the team to enable this feature.
```

I think the following line may need some rewording:

> You either need to bug them to enable it or remove them from the team to enable this feature.

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+1.  Select <label>Team</label> from the menu on the left.
+1.  Scroll down until you see the <label>Enforce two-factor authentication</label> section. There is a link to <label>Disable</label> the feature.
+
+    ![Disable Two-Factor authentication enforcement](/files/account-disable-2fa-enforcement.png)
+</div>
+
+
+## Disabling Two-Factor authentication while in an enforced team
+
+When you are part of an Two-Factor enforcing team, you can still [disable Two-Factor authentication](http://localhost:3000/articles/two-factor-authentication/#disable) on your user.
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]: Link to localhost. Also, a "When you are part of an Two-Factor enforcing team" seems more a special punishing squad :trollface:

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+<warning>
+If you are disabling Two-Factor authentication you will be removed from all accounts that enforce Two-Factor authentication.
+</warning>
+
+
+## Inviting new people into an 2FA enforcing team
+
+There are no restrictions on inviting new people to your account when you have Two-Factor authentication enforcement enabled. If the user already has a DNSimple account they can't join the team when they don't have Two-Factor authentication enabled.
+
+When you invite users to join your account that do not have an DNSimple account, they will be asked to enable Two-Factor authentication after they sign up so they can join your account.
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]: To me this one should be the first sentence. The first sentence is confusing. I also believe you used the term account for user, but they are not the same thing.

**Comment by OleMchls** [comment]
Thanks for your feedback @weppos PTAL

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+1.  Select <label>Team</label> from the menu on the left.
+1.  Scroll down until you see the <label>Enforce two-factor authentication</label> section. There is a link to <label>Disable</label> the feature.
+
+    ![Disable Two-Factor authentication enforcement](/files/account-disable-2fa-enforcement.png)
+</div>
+
+
+## Disabling Two-Factor authentication while in an enforced team
+
+When you are part of an Two-Factor enforcing team, you can still [disable Two-Factor authentication](/articles/two-factor-authentication/#disable) on your user.
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]: > "When you are part of an Two-Factor enforcing team" seems more a special punishing squad

It reads quite badly to me. It seems you are referring to an action team.

**Comment by weppos** [comment]
I'd like @itsalyse to take a look at this article and provide some help as certain sentences seems quite hard to read to me.

File: content/articles/two-factor-authentication-enforcement.markdown [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+[Two-Factor authentication](/articles/two-factor-authentication/) provides an extra level of security to protect your user credentials. You can enforce two-factor authentication on account members by enabling the feature for the account.
+
+When you have multiple users managing an account, it's vitaly important to reduce at minimum the likelyhood of an user credential being compromised. This is why this feature is available to all accounts that has the ability to [manage members of an account](/articles/account-users/).
+
+## Enabling Two-Factor authentication enforcement {#enable}
+
+You can enable Two-Factor authentication enforcement from your account members page.
+
+You can only enable Two-Factor authentication enforcement an account if all users have Two-Factor authentication enabled. You either need to bug them to enable it or remove them from the team to enable this feature, then re-invite them, they will be asked to enable Two-Factor authentication after they sign up so they can join your account.
```
Comment (weppos) [inline content/articles/two-factor-authentication-enforcement.markdown]: > You either need to bug them

This sentence here seems quite colloquial to me, not exactly something I would read in a documentation.

File: content/articles/two-factor-authentication-enforcement.markdown (hunk 8 above)
Comment (itsalyse) [reply content/articles/two-factor-authentication-enforcement.markdown]: I disbanded the action squad.

**Pull request merged**